packages/merge-on-green/package.json

```diff
@@ -28,7 +28,8 @@
   },
   "dependencies": {
     "@google-cloud/datastore": "^6.1.0",
-    "gcf-utils": "^5.5.1"
+    "gcf-utils": "^5.5.1",
+    "probot": "^9.13.0"
```

Comment:
Do we need to specifically bring in this version?  I've very much been trying to avoid diamond dependencies given our reliance here on `gcf-utils`.

Reply:
I think we need it to use it to define a type.

Reply:
Sure - but it's packaged in `gcf-utils` as a dependency, so technically you don't need to define it here.  Conventional wisdom says you are doing the right thing, but this is a bit of a weird situation.  Probot and Octokit have particularly bad diamond dependency issues.  With defining the version of probot here *and* in `gcf-utils`, we run the risk of them getting stuck on different, incompatible versions.  To ensure we only ever get a single version, we specifically don't include it in `package.json` for projects that already rely on `gcf-utils`.  Now, you will get an eslint warning.  It's ok to ignore the line where you import `probot` (like all the other bots)